cmd/minishift/cmd/util/util.go

```diff
@@ -149,3 +152,12 @@ func ValidateGenericDriverFlags(remoteIPAddress, remoteSSHUser, sshKeyToConnectR
 		atexit.ExitWithMessage(1, fmt.Sprintf("Error: %s", msg))
 	}
 }
+
+// OcClusterDown down the cluster and if there is any issue during the minishift start before even
+// oc binary cached then it just ignore the error [PK]
```

Comment:
@praveenkumar I am able to understand the comment for this function. Can you please improve the explanation?